packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx

```diff
@@ -15,37 +15,36 @@ import Calendar from './components/Calendar';
 
 export interface IDatepickerRangeProps {
   locale?: string;
```

Comment:
Please document the `locale` prop.

Reply:
Sorry I missed this prop. Fixed in latest push.